source/fab/util.py

```diff
@@ -62,6 +62,31 @@ def file_checksum(fpath):
         return HashedFile(fpath, zlib.crc32(infile.read()))
 
 
+def remove_minus_J(flags, verbose=False):
+    """
+    Remove the -J <folder> from the given flags.
+
+    """
+    # todo: what if there's two -Js ?
+    # todo: Fab should be compiler aware, with the possibly of different flags to ignore per compiler
+    try:
+        found = flags.index('-J')
+        if verbose:
+            logger.info(f"removing -J <folder> from {flags}")
+        flags = flags[:found] + flags[found+2:]
+    except ValueError:
+        pass
```

Comment:
Handle multiple `-J ITEM`?

```suggestion
    in_minus_J = False
    ret = []
    for flag in flags:
        if in_minus_J:
            in_minus_J = False
            continue
        if flag == '-J':
             in_minus_J = True
        else:
             ret.append(flag)
```

Reply:
:+1: